pi-blaster.js

```diff
@@ -31,6 +31,12 @@ var piblaster = {
 	}
 };
 
+var piblaster = {
+	releasePwm: function(pinNumber) {
```

Comment:
You should make this a part of the block above, otherwise you are overriding the other function. Can you make that change?